tasks/au/data.py

```diff
@@ -408,12 +414,30 @@ def populate_general(self):
 
             in_colnames[0] = '"{}"'.format(self._get_geoid())
 
-            cmd = 'INSERT INTO {output} ("{out_colnames}") ' \
-                  'SELECT {in_colnames} FROM {input} '.format(
-                      output=self.output().table,
-                      input=intable,
-                      in_colnames=', '.join(in_colnames),
-                      out_colnames='", "'.join(out_colnames))
+            if self.resolution in CONVERT_TO_7_DIGIT:
```

Comment:
Shouldn't this be convert_to_11_digit? It's converting from 7 to 11 digit (completing the id), right?

Reply:
Yes, I think it makes sense